tests/kit/model_zoo/transformers/llama.py

```diff
@@ -52,6 +52,9 @@ def data_gen_for_casual_lm():
                          max_position_embeddings=128,
                          num_labels=16)
 
+    if hasattr(config, "pad_token_id"):
+        config.pad_token_id = 2
```

Comment:
Why do so?

Reply:
llama2config has added this parameter. In the LlamaForSequenceClassification model's forward function, there is such a condition check: `        if self.config.pad_token_id is None and batch_size != 1:
            raise ValueError("Cannot handle batch sizes > 1 if no padding token is defined.")`

Reply:
pad_token is set to unk_token in llama2